docs/kafka/metrics-monitoring-kafka/README.adoc

```diff
@@ -86,27 +86,72 @@ As a developer or administrator, you can view metrics in {product-kafka} to visu
 
-* `kafka_topic:kafka_topic_partitions:count`: Number of topics in the cluster. This does not include internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`. The maximum number of topics is 1000.
+* `kafka_topic:kafka_topic_partitions:sum`: Number of partitions across all topics in the cluster. This does not include partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`.
++
+The Kafka instance type determines the maximum number of partitions, as shown in the following values:
 
-* `kafka_topic:kafka_topic_partitions:sum`: Number of partitions across all topics in the cluster. This does not include partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`. The maximum number of partitions is 1000.
+** 100 for a trial instance
+** 1500 for a production instance with a size of 1 streaming unit
+** 3000 for a production instance with a size of 2 streaming units
 --
 
 Broker metrics::
 +
 --
-* `kafka_broker_quota_softlimitbytes`: Maximum amount of storage, in bytes, for this broker before producers are throttled. When this limit is reached, the broker starts throttling producers to prevent them from sending additional data. The maximum storage in the broker is 1000 GiB.
+* `kafka_broker_quota_softlimitbytes`: Maximum amount of storage, in bytes, for this broker before producers are throttled. When this limit is reached, the broker starts throttling producers to prevent them from sending additional data.
++
+The Kafka instance type determines the maximum storage in the broker, as shown in the following values:
+
+** 10 GiB for a trial instance
+** 1000 GiB for a production instance with a size of 1 streaming unit
+** 2000 GiB for a production instance with a size of 2 streaming units
+
 
 * `kafka_broker_quota_totalstorageusedbytes`: Amount of storage, in bytes, that is currently used by partitions in the broker. The storage usage depends on the number and retention configurations of the partitions. This metric must stay below the `kafka_broker_quota_softlimitbytes` metric setting.
 
-* `kafka_controller_kafkacontroller_global_partition_count`: Number of partitions in the cluster. Only the broker that is the current controller in the cluster reports this metric. Any other brokers report `0`. This count includes partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`. The maximum number of partitions is 1000. This metric is similar to the `kafka_topic:kafka_topic_partitions:sum` cluster metric.
+* `kafka_controller_kafkacontroller_global_partition_count`: Number of partitions in the cluster. Only the broker that is the current controller in the cluster reports this metric. Any other brokers report `0`. This count includes partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`.
+This metric is similar to the `kafka_topic:kafka_topic_partitions:sum` cluster metric.
```

Comment:
@AedinC I'd just bring it up to follow in the previous paragraph, no line break. The non-spaced break of a line can sometimes render it all in a paragraph but sometimes not, I've found. So you can keep it here if you like the "every line a sentence" approach in the source, but that's not our norm so far. Just have a look and make sure it looks good in the rendering. Thanks.

Reply:
Thanks @sterobin. I'll change that.